projects/igniteui-angular/src/lib/core/styles/components/_index.scss

```diff
@@ -8,7 +8,6 @@
 @forward 'card/card-theme';
 @forward 'calendar/calendar-theme';
 @forward 'carousel/carousel-theme';
-@forward 'charts';
```

Comment:
We still need to forward the chart themes. But this needs to be done from the theming package this time around.